scripts/compare_noupdate_xml_records.py

```diff
@@ -185,12 +200,13 @@ def main(argv=None):
         if len(element):
             data.append(element)
 
-    openerp = etree.Element("openerp")
-    openerp.append(data)
-    document = etree.ElementTree(openerp)
+    odoo = etree.Element("odoo")
+    odoo.append(data)
```

Comment:
This is when we are preparing the output.

The case u mentioned has already been considered, pls check: `process_record_node`. :)